marimo/_ast/app.py

```diff
@@ -54,6 +54,7 @@ class _AppConfig:
     a single marimo program.
     """
 
+    # TODO(akshayka): rename normal -> compact
```

Comment:
could make the breaking change later, but to stop the bleeding, could add `compact` to the list, but still allow `normal` as a valid value